Resources/Server Info/SimpleStation14/Guidebook/AlertProcedure/StandardOperatingProcedure.xml

```diff
@@ -27,26 +27,29 @@ If they are unfit for duty the Head of Personnel may demote them to assistant. I
-- Military Explosives when not in use by Security or otherwise authorised entities. - Note that constructive explosives, such as any charges used for mining are not subject to this restriction. Plasma-based explosives are considered Military Weaponry.
-- Automatic Weaponry when not in use by Security, another authorised department of Central Command, or an otherwise authorised entity. - This means any weapon capable of firing multiple shots when the trigger is depressed once, energy or otherwise.
-- Corporate Security forces are considered an 'authorized entity' with repsect to the above clauses.
-- Weaponry in the possession of personnel who have not been authorised to possess it.
-- Hacking Software or Hardware which allows a person to subvert access to ship systems, bypass access restrictions or attack a vessels computer network.
-- Any hostile corporation technology.
+- Space Drugs
+- Chloral Hydrate, specifically any dosage above 15u, when not prescribed by the Psychologist, no matter the case
+- Pure Lysergic Acid Diethylamide, (LSD) sometimes referred to as 'MindBreaker'
+- Pure Ephedrine, when not in the possession of the CMO or Security
+- Pure DesoxyEphedrine, commonly referred to as 'Meth'
+- Any explosive or pyrotechnic compound outside of the Science department testing area
+- Any lethal toxin or toxic substance/compound outside of the Medical chemistry lab
+- Any gases that are not normally found in the atmosphere of the station or station's planet, when not in possession of the CE or Atmospheric Technician
+- Liquid Miasma, when not in possession of the CMO or a Virologist for research purposes only
+- Any Reagents made mainly by hostile corporations
+- XenoBlood, or the corpse of a Xenomorph
+- Corpium
+- Nocturine
+
+Controlled Items are items that have been restricted or made illegal under Space Law. Corporate Security forces are considered an 'authorized entity' with repsect to the listed clauses. The current list of controlled items is as follows:
+- Any rapid-injection technology, such as chemipens, hyposprays, or autoinjectors, when not in possession of or given by the CMO or a Medical Doctor
+- Biological or Chemical Weaponry, such as neurotoxin grenades or listed controlled substances. Plasma gas is considered a chemical weapon when used offensively
+- Any cloning technology designed to rapidly grow an adult clone with fully-developed mental faculties not in possession of the medical department
+- Military Explosives when not in use by Security or otherwise authorised entities. Note that constructive explosives, such as any charges used for mining are not subject to this restriction. Plasma-based explosives are considered Military Weaponry
+- Automatic Weaponry when not in use by Security, another authorised department of Central Command, or an otherwise authorised entity. This means any weapon capable of firing multiple shots when the trigger is depressed once, energy or otherwise
```

Comment:
```suggestion
- Automatic Weaponry when not in use by Security, another authorised department of Central Command, or an otherwise authorised entity. This refers to any weapon capable of firing multiple shots when the trigger is depressed once, energy or otherwise
```
Picky :P